application/admin/views.py

```diff
@@ -210,3 +210,15 @@ def data_sources():
     data_sources = DataSource.query.order_by(DataSource.title).all()
 
     return render_template("admin/data_sources.html", data_sources=data_sources)
+
+
+@admin_blueprint.route("/data-sources/merge")
+@login_required
+@user_can(MANAGE_USERS)  # TODO: update to MANAGE_DATA_SOURCES
+def merge_data_sources():
+
+    data_source_ids = request.args.get("ids").split(",")
+
+    data_sources = DataSource.query.filter(DataSource.id.in_(data_source_ids))
```

Comment:
I'd probably prefer to fail fast if any of the data source IDs are not found/valid. Maybe raise a 400 in that case?

Reply:
Done in afa49e0ae7ed03a0af507acc413d941783339960